code_annotations/cli.py

```diff
@@ -96,15 +98,17 @@ def django_find_annotations(
                 click.echo("Coverage passed without errors.")
 
             if report:
-                searcher.report(annotated_models)
+                searcher.report(annotated_models, app_name)
 
             annotation_count = 0
 
             for filename in annotated_models:
                 annotation_count += len(annotated_models[filename])
 
             elapsed = datetime.datetime.now() - start_time
-            click.echo("Search found {} annotations in {}.".format(annotation_count, elapsed))
+            click.echo("Search found {} annotations in {} seconds.".format(
+                annotation_count, elapsed.total_seconds()
+            ))
```

Comment:
Currently, this output looks like this:
`Search found 357 annotations in -1 day, 19:00:12.574411.`
This change will make it look like this:
`Search found 357 annotations in 29.3801 seconds.`

Reply:
Thanks for this. I have a different fix for it in the big refactor PR, but I'm glad to see it sorted.